lib/l10n/app_es.arb

```diff
@@ -112,6 +112,8 @@
   "editComment": "Editar comentario",
   "comment": "Comentario",
   "comments": "Comments",
+  "visitInstance": "Visit Instance",
```

Comment:
Done in e3978d90f69d5c2cecd3af11717cf7da8ea8e14d.

Reply:
I would same the same applies to `lib/l10n/app_pl.arb` as well! Also, since `sv.arb` and `fi.arb` files have been removed, we can also remove them from there too.

Reply:
Oops, sorry! Should be all set now.